### ISSUE TITLE
Cannot read file '../tsconfig.json'

### ISSUE THREAD

**Issue by slubimav** [remix-run/grunge-stack#61]
### Have you experienced this bug with the latest version of the template?

Yes

### Steps to Reproduce

npx create-remix --template remix-run/grunge-stack

Where would you like to create your app? **remix-blog**
Do you want me to run `npm install`? (Y/n) **Y**
TypeScript or JavaScript? **JavaScript**
Do you want to run the build/tests/etc to verify things are setup properly? **Yes**

`error TS5083: Cannot read file '/Users/Siarhei_Liubimau/Documents/remix/remix-blog/tsconfig.json'.`

### Expected Behavior

The template should work out of the box.

### Actual Behavior

The template is broken and unusable.

**Comment by mcansh** [comment]
hey! do you have an empty `tsconfig.json` file in the parent directory by chance?

**Comment by 47Key** [comment]
I found that going through the config files and changing the extensions from .ts to .js worked for me, the issue seems to be coming from cypress. I deleted the tsconfig file in the cypress folder, and removed the eslint export file that included the tsconfig file.

the vitest.config, remix.config and cypress.config files all had .ts file extensions listed. Deleting any mention of typescript files in the config files seemed to fix the issues. After doing that running "npm run dev" brings me to the default grunge stack homepage. It's a bit buggy but I wouldn't consider it unusable, just a bit of digging around to find the problem.
